bin/install-app.bat

```diff
@@ -0,0 +1,30 @@
+@echo off
+REM This program and the accompanying materials are
+REM made available under the terms of the Eclipse Public License v2.0 which accompanies
+REM this distribution, and is available at https://www.eclipse.org/legal/epl-v20.html
+REM 
+REM SPDX-License-Identifier: EPL-2.0
+REM 
+REM Copyright Contributors to the Zowe Project.
+
+if [%1]==[] goto :fail
+setlocal
+set app_path="%~f1"
+cd "%~dp0..\..\zlux-server-framework\utils"
+node unpackage-app.js -i "%app_path%"  -o "%~dp0..\..\\" -p "%~dp0..\..\zlux-app-server\deploy\instance\ZLUX\plugins" %2
```

Comment:
This looks like this assumes that the unpacked plugin should go into the same directory as Zowe. While that may be a decent default, I think that the bat/sh scripts should take an optional argument to pass to -o

Also, I'm wondering about the -p argument: isn't the location of the "plugins" directory determined by the "pluginsDir" field of zluxserver.json? So, shouldn't this script take the location of zluxserver.json (defaults to %~dp0..\..\zlux-app-server\deploy\instance\ZLUX\serverConfig\zluxserver.json) as another optional argument, then use the contents of the config to determine where to place the plugin locator file?

Reply:
Good point, it's better to use -c rather than -p for slightly better flexibility. These scripts aren't too flexible but that will give the ability to have a different plugins location.